app/(gcforms)/[locale]/(form administration)/form-builder/support/[[...supportType]]/page.tsx

```diff
@@ -29,7 +29,7 @@ export default async function Page({
 
   return (
     <DefaultLayout showLanguageToggle>
-      <ClientSide supportType={supportType ? supportType[0] : ""} />
+      <ClientSide supportType={supportType ? supportType[0] : "support"} />
```

Comment:
so we introducing a default?

Reply:
Sort of : ) I suppose the default is undefined in which case a not found is shown.
This code originally made more sense to be combined and handle both support and contact us "pages. It's changed over time though and IMO has become difficult to maintain. I'd like to refactor this out into a separate support and a separate contact us page in the future.